addons/website_event_track/tests/test_website_tour.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# -*- coding: utf-8 -*-
+
+from datetime import datetime, timedelta
+
+from odoo import http
+from odoo.tests import tagged
+from odoo.tests.common import HttpCase, TransactionCase
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteTour(HttpCase, TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.env['res.config.settings'].write({
+            'module_website_event_track_live': True,
+        })
+        event_date_begin = datetime.now()
+        self.event = self.env['event.event'].create([{
+            'name': 'Test event',
+            'date_begin': event_date_begin,
+            'date_end': event_date_begin + timedelta(weeks=1),
+            'is_published': True
+        }])
+        self.env['event.track'].create([{
+            'name': 'Introduction class',
+            'date': event_date_begin + timedelta(hours=1),
+            'date_end': event_date_begin + timedelta(hours=2),
+            'event_id': self.event.id,
+            'is_published': True
+        }])
+
+    def _get_email_reminder_tour_url(self):
+        return f'{self.event.website_url}/agenda'
```

Comment:
I think that we can pass on this one, it doesn't simplify much and makes it a bit harder to read :slightly_smiling_face: